app/models/bulkrax/csv_entry_decorator.rb

```diff
@@ -0,0 +1,19 @@
+module Bulkrax
+  # Override Bulkrax v4.4.0
+  #
+  # @see https://github.com/samvera-labs/bulkrax/pull/719 Remove override when this PR is part of
+  #      this application's Bulkrax.
+  module CsvEntryDecorator
+    def establish_factory_class
+      parser.model_field_mappings.each do |key|
```

Comment:
Would this mean that the work_type and/or model column would no longer be required @jeremyf

Reply:
It doesn't change whether or not it's required.  It ensures that we first set the underlying `factory_class` which helps govern what fields we accept or reject in the import process.